test/circuit_cutting/test_cutting_reconstruction.py

```diff
@@ -12,13 +12,17 @@
 import unittest
 from ddt import ddt, data, unpack
 
+import pytest
 import numpy as np
+from qiskit.result import QuasiDistribution
 from qiskit.quantum_info import Pauli, PauliList
 from qiskit.circuit import QuantumCircuit, ClassicalRegister
 
 from circuit_knitting_toolbox.utils.observable_grouping import CommutingObservableGroup
+from circuit_knitting_toolbox.circuit_cutting.qpd import WeightType
 from circuit_knitting_toolbox.circuit_cutting.cutting_reconstruction import (
-    process_outcome,
+    _process_outcome,
```

Comment:
I made process_outcome and append_circuit_measurements private for 0.2.0, as that is the style I used in the rest of the modules. I wanted to keep all helper functions private until it was clear they would be a value-add to the API.

I want to keep the initial API as light as possible

Reply:
Yes, I saw this, and I agree this makes most sense.